Woof/WoofTests/PackageIntegration/NetworkServiceIntegrationTests.swift

```diff
@@ -0,0 +1,8 @@
+import NetworkService
```

Comment:
I'd suggest to prepare more flexible structure and use the `Woof/WoofTests/ThirdParty/NetworkService /NetworkServiceIntegrationTests.swift ` path instead of `Woof/WoofTests/PackageIntegration/NetworkServiceIntegrationTests.swift`.

Reply:
Fixed https://github.com/ios-course/ironfoudation-team-project/pull/188/commits/ad70909a2b4d7ffedf05b4e3a7927677fb31e345